.github/workflows/interface-unit-tests.yml

```diff
@@ -375,12 +376,53 @@ jobs:
         ${{ inputs.additional_python_packages }}
       additional_pip_packages_post: ${{ needs.default-dependency-versions.outputs.pennylane-lightning-latest }}
       pytest_coverage_flags: ${{ inputs.pytest_coverage_flags }}
-      pytest_markers: jax and not qcut and not finite-diff and not param-shift
+      pytest_markers: jax and not qcut and not finite-diff and not param-shift and not capture
       pytest_additional_args: --dist=loadscope --splits 4 --group ${{ matrix.group }} ${{ needs.warnings-as-errors-setup.outputs.pytest_warning_args }}
       pytest_durations_file_path: '.github/durations/jax_tests_durations.json'
       pytest_xml_file_path: '${{ inputs.job_name_prefix }}jax-tests (${{ matrix.group }}, ${{ matrix.python-version }})${{ inputs.job_name_suffix }}.xml'
       requirements_file: ${{ github.event_name == 'schedule' && strategy.job-index == 0 && 'jax.txt' || '' }}
 
+  capture-jax-tests:
+    needs:
+      - setup-ci-load
+      - determine_runner
+      - default-dependency-versions
+      - warnings-as-errors-setup
+    strategy:
+      fail-fast: ${{ needs.warnings-as-errors-setup.outputs.fail_fast == 'default' }}
+      max-parallel: >-
+        ${{
+           fromJSON(needs.setup-ci-load.outputs.matrix-max-parallel).capture-jax-tests
+           || fromJSON(needs.setup-ci-load.outputs.matrix-max-parallel).default
+         }}
+      matrix:
+        group: [1]
```

Comment:
Do we need this, since it's just one group?

Reply:
🤷‍♀️ Does it hurt? And might make it easier to add another runner if we ever need one.

Reply:
We would need to add a note to update the `--splits` to be the length of this matrix if it does get updated. Otherwise, we either hit errors or have multiple workers execute the same tests